tests/test_integration.py

```diff
@@ -0,0 +1,33 @@
+from pathlib import Path
+import subprocess
+import json
+import pytest
+from typing import Dict, List, Tuple
+
+this_dir = Path(__file__).parent.resolve()
+fixtures_dir = this_dir / 'fixtures'
+exe_path = this_dir / '..' / 'build' / 'submitty_count_ts'
+
+def execute_command(file: str, command: str) -> int:
+    args = [exe_path]
+    args.extend((f"-l {file.split('.')[1]} {command} {str(this_dir / 'fixtures' / file)}").split(" "))
```

Comment:
Is there a better way to build this argument list?  Doing it like this prevents us from testing with files which have multiple `.` characters or whitespace in the filename in the future.

Something like this would be better I think:
`args = [exe_path, '-l', file.split('.')[-1], command, str(this_dir / 'fixtures' / file)]`

Reply:
Good point, done in 877c3467b5fea8d60a39121c67e8f80a4e3c83a4 + 2fdbf5f8bb61d1dde4a76f7f6991633345dbe5c3

Reply:
Ended up tweaking the suggestion some as `command` as it was needed to still be split for space, so I've done that within `commands.json` file, which should allow for a space to be used for the countable as well as feature if we ever need that, though not sure we will.